tests/integration/test_remove_relation.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
There's already a file called integration.py, please add new tests there instead of creating a new file.

Reply:
done